stable_baselines/common/vec_env/base_vec_env.py

```diff
@@ -239,8 +240,11 @@ def getattr_recursive(self, name):
         :param name (str) name of attribute to look for
         :return: (object) attribute
         """
-        if name in self.__dict__:  # attribute is present in this wrapper
-            attr = self.__dict__[name]
+        class_attributes = {k: v for k, v in inspect.getmembers(self.__class__)}
```

Comment:
why not do that in the `__init__` ?

Reply:
Indeed, no need to repeat this with every call.